lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -566,7 +566,9 @@ void OpenCLFunction::execute() {
 
       if (auto *SI = dyn_cast<SplatInst>(&I)) {
         // Pass the splat as a parameter.
-        setKernelArg(kernel, ++numArgs, SI->getValue());
+        if (!isQuantized) {
```

Comment:
would be easier to follow if you handle both quantized and non-quantized splat here.

Reply:
changed